src/routes/MemberProfile/MemberProfile.js

```diff
@@ -226,10 +226,11 @@ export function ActionDropdown ({ items }) {
     />
 }
 
-export function Project ({ memberCap, project, showDetails }) {
-  const { title, id, createdAt, creator, members } = project
+export function Project ({ memberCap, project, routeParams, showDetails }) {
+  const { title, id, communities, createdAt, creator, members } = project
+  const postTypeContext = communities.map(c => c.slug).includes(routeParams.slug) ? 'project' : undefined
```

Comment:
i dont really understand what this is doing. what happens if the project does not exist in the current communities?

Reply:
a comment would help